### DISCUSSION TITLE
skywalking miss trace with large  requests，which parameter should be set or expand

### DISCUSSION THREAD

**Discussion by liyuntian** [apache/skywalking#10235] [General]
my environment is 
5  skywalking-oap-es7  nodes each  with 8 core 16Gi  JAVA_OPTS is -Xms10g -Xmx10g -Xmng

I use jmeter to test my client with 500 threads, and I get summary report with HTTP Request 11980 sample
but I only get 10005 trace in skywalking ui,  skywalking or es7 lost some data? which parameters should be expand?
thanks

**Comment by wu-sheng** [comment]
Can't tell with this context. Agent has fail-safe mechanism to keep your service as stable as possible, rather than OOM. So, if your backend can't process the traces in time, the agent would abandon traces automatically.

**Reply by liyuntian** [reply]
Our production environment http request is 12000qps, 1 trace with 10 spans , is there best practice parameters setting，or any suggestions on parameter configuration ? now we are running performance tests， any suggestion is appreciated. 
thanks

**Reply by wu-sheng** [reply]
This doesn't look like a large traffic to me. 
There is no simple answer. I don't know what to suggest.

**Reply by liyuntian** [reply]
I test 150 million request data, which is about 70Gb. and 12000 qps request , which is about 500G a day. For me, it is quite large. I haven't encountered so much data. Maybe some of the skywalking Oap parameters I configured are wrong. I continue to optimize them
thank you

**Reply by wu-sheng** [reply]
I think you treat a community in the wrong way. This is not customer service. If you have tech issues, we may have some suggestions.
But we can't and wouldn't give any feedback based on your local and private testing.

This is not how we discuss in an open-source community.

**Reply by liyuntian** [reply]
Thank you for your guidance
